packages/katello/katello/backup.rb

```diff
@@ -324,7 +324,18 @@ def generate_metadata
     end
 
     def create_pulp_data_tar
-      run_cmd("tar --selinux --create --file=#{File.join(@dir, 'pulp_data.tar')} --exclude=var/lib/pulp/katello-export --listed-incremental=#{File.join(@dir, '.pulp.snar')} --transform 's,^,var/lib/pulp/,S' -S *")
+      archive_size = @options[:split_tar]
+
```

Comment:
this is nitpicky (sorry!) but you can do without as much whitespace here and below. I think just a space around the top and bottom of the if statement block is ok, same for the restore part as well